backends/ebpf/targets/ebpfenv.py

```diff
@@ -16,155 +16,157 @@
     interfaces. The bridge runs in a completely isolated namespace.
     Allows the loading and testing of eBPF programs. """
 
-import os
 import sys
-from subprocess import Popen, PIPE
+import logging
+from pathlib import Path
 
-sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)) + '/../../tools')
-from testutils import *
+# Append tools to the import path.
+FILE_DIR = Path(__file__).resolve().parent
+# Append tools to the import path.
+sys.path.append(str(FILE_DIR.joinpath("../../../tools")))
+import testutils
 
 
-class Bridge(object):
+class Bridge:
```

Comment:
I think there is some difference in behaviour of classes that are derived explicitly from `object`. You should probably check that, maybe there is a reason for that to be like it was.

Reply:
A lot of this code was written for Python2, which at some point required object iirc.